Add support for identification via health insurance card

fkztw opened the pull request:
+ Add install files of Health Insurance ID Card plugin
+ Install plugin for health insurance card in Dockerfile
+ Open new tab for health insurance card in firefox
+ Start plugin for health_insurance_card when start container
+ Add health insurance card related content into README
+ Add Vim swap files into gitignore
+ Remove trailing spaces in Dockerfile
+ Rename firefox.sh to start.sh

Commit:
Add install files of Health Insurance ID Card plugin.

Downloaded from:
<https://cloudicweb.nhi.gov.tw/cloudic/system/SMC/mEventesting.htm>

Commit:
Make status_of_proc can be used in docker container.

For checking status of mLNHIICC.

Commit:
Install plugin for health insurance card in Dockerfile.

Commit:
Open new tab for health insurance card in firefox.

Commit:
Add health insurance card related content into README

Commit:
Add Vim swap files into gitignore

Commit:
Start plugin for health_insurance_card when start container.

Commit:
Remove trailing spaces in Dockerfile

Commit:
Rename firefox.sh to start.sh

To make it more suitable for what it does because
it not only starts firefox.

Commit:
Remove extracted files of mLNHIICC_Setup

Just use Docker ADD health_insurance_card/mLNHIICC_Setup.Ubuntu16.tar.gz
to make git more clear

chihchun commented on 2017/Dockerfile:
```diff
@@ -36,12 +36,13 @@ RUN chmod 755 /usr/local/bin/start.sh
 
-# Install script for plugin of Health Insurance ID Card will use killall command
-COPY health_insurance_card /tmp/health_insurance_card/
+ADD health_insurance_card/mLNHIICC_Setup.Ubuntu16.tar.gz /tmp/health_insurance_card/
```
Is there a download link for mLNHIICC_Setup.Ubuntu16.tar.gz? I don't feel like redistribute the binary blobs.

fkztw replied:
I've fixed this problem in https://github.com/chihchun/personal-income-tax-docker/pull/1/commits/aa4cb1794379cec35c5649a01d4787bb2add2a19.
Please have a look.
Thanks.

chihchun reviewed (changes requested):
Hi, I preferred to download the zip file during build process, but not preloaded into the repository.

Commit:
Get mLNHIICC_Setup.Ubuntu.zip from website directly

Do not download it and save it into git repository.

Because of the certificate of https://cloudicweb.nhi.gov.tw is not
valid, I have to use `wget --no-check-certificate` to download it.
